synapse_data_warehouse/synapse_event/tasks/V2.54.2__create_processedaccess_event_task.sql

```diff
@@ -0,0 +1,15 @@
+use database {{database_name}}; --noqa: JJ01,PRS,TMP
+
+-- Step 1) Pause the upstream tasks
+alter task synapse_raw.refresh_synapse_warehouse_s3_stage_task suspend;
+alter task synapse_raw.processedaccess_task suspend;
+
+-- Step 2) Create the task of interest
+create or replace task synapse_raw.create_access_event_task
```

Comment:
@philerooski Almost forgot - I introduced the new tasks into `synapse_raw` rather than `synapse_event`. Would it make more sense for me to move these to `synapse_event`? If I switch over the `synapse_event` I would need to manually grant ownership of them to the PROXY_ADMIN role, right?

Reply:
I'll give it a go on the first try and ping you if there are issues!

Reply:
I used [this script](https://github.com/Sage-Bionetworks/snowflake/blob/dev/admin/ownership_grants/V1.7.3__internamespace_object_ownership_prod.sql) as the blueprint for managing the task grants on `synapse_event`. We'll see if it works!

BTW where is your final doc that summarized the RBAC system and all the nuances we agreed on? For some reason I can't find it on the [design docs](https://sagebionetworks.jira.com/wiki/spaces/DPE/pages/2733867066/Design+Documents) page

Reply:
:o a new Snowflake nuance we have discovered:

```
snowflake.connector.errors.ProgrammingError: 091413 (42601): Cannot have predecessor FILEUPLOAD_TASK from a different schema.
```

Reply:
Ugh, that is very annoying. I also found that specified in the docs for the `AFTER` parameter: https://docs.snowflake.com/en/sql-reference/sql/create-task

Perhaps we can lean into this Snowflake design choice: let's put the task in the `synapse_raw` schema. I believe we already have tasks which exist in the `synapse_raw` schema but which move data into the `synapse` schema, so this is not entirely unprecedented.

Reply:
@philerooski seems like something not in our control, but I recommend you asking the #sage-snowflake channel so our snowflake support engineer can answer the quesiton.